packages/admin-panel/src/VizBuilderApp/components/DashboardItem/DashboardItemMetadataForm.jsx

```diff
@@ -68,18 +82,31 @@ export const DashboardMetadataForm = ({ Header, Body, Footer, onSubmit }) => {
           inputRef={register({
             required: 'Required',
           })}
-          value={searchInput}
+          value={permissionGroupSearchInput}
           onInputChange={(event, newValue) => {
-            setSearchInput(newValue);
+            setPermissionGroupSearchInput(newValue);
           }}
         />
+        <Autocomplete
+          id="vizType"
+          name="vizType"
+          label="Visualisation Type"
+          placeholder="Select Visualisation Type"
+          defaultValue={vizTypeOptions.find(({ value }) => value === vizType)}
+          options={vizTypeOptions}
+          getOptionLabel={option => option.label}
+          getOptionSelected={option => option.value}
+          inputRef={register({
+            required: 'Required',
+          })}
```

Comment:
I noticed that this is required but if you try to enter without completing it doesn't give you any feedback for this field

Reply:
Ah nice catch, fixed